pandas/io/formats/latex.py

```diff
@@ -58,10 +58,9 @@ def write_result(self, buf: IO[str]) -> None:
         """
         # string representation of the columns
         if len(self.frame.columns) == 0 or len(self.frame.index) == 0:
-            info_line = "Empty {name}\nColumns: {col}\nIndex: {idx}".format(
-                name=type(self.frame).__name__,
-                col=self.frame.columns,
-                idx=self.frame.index,
+            info_line = (
+                f"Empty {type(self.frame).__name__}\n"
+                f"Columns: {self.frame.columns}\nIndex: {self.frame.index}"
```

Comment:
can you break this at the \n for readability

Reply:
Sure